tests/std/tests/P0220R1_optional/test.cpp

```diff
@@ -1497,7 +1497,7 @@ void test_with_test_type() {
     assert(T::alive == 0);
 }
 
-void test_ambigious_assign() {
+void test_ambiguous_assign() {

```

Comment:
No change requested here, but this test for `optional` is primarily an aggregation of the libc++ tests, so this misspelling of "ambiguous" comes from LLVM. I'd appreciate it if you could also fix this [upstream](https://github.com/llvm/llvm-project) so we don't reintroduce the error the next time we update.

Reply:
I'll try to get to that by this weekend.

Reply:
Um... well... um... [llvm](https://gist.github.com/jsoref/ca0f501e19a100db694a47c0646c3d07) is going to take a while.

I'm at `2020-04-24T11:08:53.0031077Z` working to `2020-04-24T11:08:54.3126823Z`.

That might not seem like a lot, but consider that the [run](https://github.com/jsoref/llvm-project/actions/runs/86511665) took a bit over 3 hours.
Whereas STL [runs](https://github.com/jsoref/STL/actions) take 3 1/2 to 4 minutes. And the output can include ~10 lines per millisecond. -- And this isn't fixing things, it's just identifying candidates for fixing.

I might need help approaching them with a PR. If someone here is an active contributor...
Projects tend to be fairly open to PRs w/ <20 changes. But GitHub complains when there are ~300 commits (I can easily hit 500 commits for some repositories -- but, I'm already potentially over 100 commits and I'm not done w/ the `a`s).

Reply:
I've pushed a spelling correction to LLVM for only "ambigious" to correct this local problem. Feel free to continue fighting the good fight there, but this is now unblocked.